app/models/authentication_provider/keycloak.rb

```diff
@@ -3,7 +3,6 @@ class AuthenticationProvider::Keycloak < AuthenticationProvider
   self.oauth_config_required = true
 
   validates :realm, presence: true
-  validates :realm, format: { without: /\s/ }
-
+  validates :realm, format: { with: URI.regexp(%w(http https)), allow_blank: true, message: :invalid_url }
```

Comment:
Why `presence: true` but `allow_blank: true`?

Reply:
yeah, probably not needed, this was a copy-paste from another place.